cli/azd/pkg/tools/python/python.go

```diff
@@ -118,10 +128,25 @@ func (cli *PythonCli) CreateVirtualEnv(ctx context.Context, workingDir, name str
 	return nil
 }
 
-func pythonExe() string {
+func checkPath() (string, bool, error) {
+	var found bool
+	var err error
 	if runtime.GOOS == "windows" {
-		return "py" // https://peps.python.org/pep-0397
+		// py for https://peps.python.org/pep-0397
+		pyString := [2]string{"py", "python"}
+
+		for _, py := range pyString {
+			found, err = tools.ToolInPath(py)
+			if found && err == nil {
+				return py, found, nil
+			}
+		}
+		return "", found, err
 	} else {
-		return "python3"
+		found, err := tools.ToolInPath("python3")
```

Comment:
Let's do the same for linux and use ["python3", "python"] in that order.
That would also allow folks using devcontainers to find python... as it is installed as a feature using `python-is-python3`